server/src/controllers/answers.controller.js

```diff
@@ -28,6 +28,20 @@ const createAnswers = (req, res) => {
     });
 };
 
+const deleteAnswer = async(req, res) => {
+    try{
+
+        const answer = await Answers.findOneAndDelete({_id: req.params.id, createdBy: req.user._id})
```

Comment:
I think `_id` is every time unique so no need to find by `createdBy`

Reply:
@mitalirs can you check it?

Reply:
actually, checking the createdBy field ensures that the person who is trying to delete that particular answer is the same as the person who created the answer. Like, if a person A creates an answer, it will have a unique _id property but if B tries to delete that answer he shouldn't be able to do so.

Reply:
@mitalirs Thanks for clarification :)